### ISSUE TITLE
Let us choose TVDB ordering please

### ISSUE THREAD

**Issue by raidenlal** [Sonarr/Sonarr#7023]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

I was (again) going to look at TVDB why and how are they ordering futurama episodes and discovered on their site:

"Usable as "Alternate Order" in API-connected systems. "

Basically: https://github.com/Sonarr/Sonarr/issues/255 and one comment in particular: https://github.com/Sonarr/Sonarr/issues/255#issuecomment-142949276 

But the issue is locked and it is mentioned that TVDB does not support this. They say they do now. Any chance of progress? 

### Describe the solution you'd like

Drop down menu for each series for which naming/ordering to be used.

Preferably if it could even do automatic rearrangement of seasons episodes (same as "rename episodes" does). But this seems like a stretch. 

### Describe alternatives you've considered

N/A

**Comment by augustuen** [comment]
Duplicate of #6499, #4136 , #3937, #321, and #255.

If you actually looked at #255, you'd see that the "blocked by skyhook" label has been removed, and the issue was assigned to one of the devs a couple weeks ago. Progress is being made, but it takes time.

**Comment by markus101** [comment]
Regardless of it being locked the issue is still open and still visible to us. Don't open new issues to ask questions about other issues.
